software-modules/langs/java/javac/java.sh

```diff
@@ -36,8 +35,50 @@ savechanges /tmp/$NAME.hsm
 ## Clean package to maintain only relevant files
 hsm2dir /tmp/$NAME.hsm
 cd /tmp/$NAME.hsm
-find . ! -path "./usr/lib/jvm*" ! -path "./usr/bin*" ! -path "./usr/share/icons*" ! -path "./usr/share/man*" ! -path "./usr/share/pixmaps*" ! -path "./etc/alternatives*" ! -path "./usr/share/application-registry*" ! -path "./usr/share/lintian*" ! -path "./usr/share/applications/java-documentation.desktop" ! -path "./usr/share/doc/reference/java*" -delete
+
+# Array to store paths to exclude that contain java things
+declare -a EXCLUDE_PATHS=(
+    "./etc/.java/*"
+    "./etc/alternatives/*"
+    "./etc/java-11-openjdk/*"
+    "./etc/java-17-openjdk/*"
+    "./etc/ssl/*"
+    "./usr/bin/*"
+    "./usr/lib/*"
+    "./usr/lib/jvm/*"
+    "./usr/sbin/update-java-alternatives*"
+    "./usr/share/applications/java-documentation.desktop*"
+    "./usr/share/bash-completion/completions/update-java-alternatives*"
+    "./usr/share/ca-certificates-java/*"
+    "./usr/share/doc/*"
+    "./usr/share/gdb/auto-load/usr/lib/jvm/*"
+    "./usr/share/gtksourceview-4/language-specs/*"
+    "./usr/share/java/*"
+    "./usr/share/lintian/*"
+    "./usr/share/man/man1/*"
+    "./usr/share/man/man8/*"
+    "./usr/share/maven-repo/*"
+    "./usr/share/mime/*"
+    "./usr/share/nano/*"
+    "./usr/share/netbeans/platform18/*"
+    "./usr/share/nvim/runtime/*"
+    "./usr/share/source-highlight/*"
+)
+
+# Construct the find command with the paths to exclude
+FIND_CMD="find ."
+for PATH in "${EXCLUDE_PATHS[@]}"; do
+    FIND_CMD+=" ! -path \"$PATH\""
+done
+FIND_CMD+=" -delete || true"
+
+# Execute the find command
+eval "$FIND_CMD"
```

Comment:
I've test it and I'm not being able to make this work. Are you removing the `set -xe` when testing in your side?
huronOS should act as a container, so in theory what I'm testing should be the exact same environment as you. Are you testing with a build made on top of your PR? Or you are testing in other distro?
I'm not really sure how is that this worked for you and not for me, but I'm open if I'm doing anything wrong to test this.

The errors I'm getting are: 
```terminal
$ eval find . '!' -path '"./etc/.java/*"' ....... -delete
./java.sh: line 76: find: command not found
$$ true
$ cd ..
$ dir2hsm /tmp/javac.hsm
./java.sh: line 81: dir2hsm: command not found
```

I'm not really sure of what's happening as I do have the command on my path, so my guess is that eval is changing the path somehow.

Reply:
Ok, I've figured out the problem with the PATH variable. Please I'm interested on how did you tested this PR, I don't think this would work in any linux-like system.